card/mod/05_standard/set/self/admin_info.rb

```diff
@@ -0,0 +1,26 @@
+view :core do |args|
+  warning = ''
+  if Card.config.action_mailer.perform_deliveries == false
+    warning += "<li class='list-group-item list-group-item-warning'>Email delivery is turned off. Change settings in config/application.rb to send sign up notifications.</li>"
+  end
+  if Card.config.recaptcha_public_key == Card::Auth::DEFAULT_RECAPTCHA_SETTINGS[:recaptcha_public_key]
+    warning +=  "<li class='list-group-item list-group-item-warning'>" + process_content(
```

Comment:
Line is too long. [89/80]<br>Operator `+=` should be surrounded with a single space.